include/ndarray/pybind11.h

```diff
@@ -42,13 +42,131 @@
  */
 
 #include "pybind11.h"
+#include "pybind11/numpy.h"
 
-#include "Python.h"
 #include "ndarray.h"
-#include "ndarray/converter/numpy.h"
-#include "ndarray/converter/ufunctors.h"
-#include "ndarray/converter/Vector.h"
-#include "ndarray/converter/eigen.h"
+#include "ndarray/eigen.h"
+#include "ndarray/converter/PyManager.h"
+
+namespace ndarray {
+
+#if PYBIND11_VERSION_MAJOR == 2 && PYBIND11_VERSION_MINOR <= 1
+using pybind11_np_size_t = size_t;
+#else
+using pybind11_np_size_t = ssize_t;
+#endif
```

Comment:
I assume we can't just get away with using `auto` and have the compiler worry about this?

Reply:
No, because at one point we need to initialize some arrays of this type.